### PR TITLE
[#23] Migration and updated data models for User/Organization changes

### PR DESCRIPTION
**Pull request by francisli**

### CODE REVIEW

**Comment by francisli** [comment]
@julianmclain @komerela 

This PR updates the data model classes, fixtures, and tests for the latest changes from @billhoke.

Note that, because of the significant structural changes, and the fact that we don't actually have any live data, the "migration" in this case is just dropping the existing schema and re-creating it entirely, rather than modifying it, which is what we'd normally do to preserve existing data. This was just faster and easier in this early particular case...

So, when you run the migration, the db will be completely empty, and you can re-run the `sequelize db:seed:all` to create the initial org/user. And I've updated the bin/create-user script as well.

**Comment by francisli** [comment]
@julianmclain if you can review and approve, I'll get it merged into master and then down into your API branch...

**Comment by julianmclain** [comment]
@francisli This looks great! The upgraded create-user script will be nice, and it's cool to see how you setup the createdBy / updatedBy aliases.

File: test/helper.js [hunk 1]
```diff
@@ -7,10 +7,19 @@ const path = require('path');
...
+  await models.sequelize.transaction(async (transaction) => {
+    /// special case handling: create the first org and superuser manually, due to the circular references
+    // const org = await models.Organization.create({
+    //   id: 'f19c4a0d-227c-4929-a223-5da89f7c1d52',
+    //   name: 'Code for San Francisco',
+    //   type: 'C4SF',
+    // });
+    // const superuser = await models.User.create({
+    //   id: 'df08937c-3db7-45f6-906c-d5226c56f4f6'
+    // });
```
Comment (julianmclain) [inline test/helper.js:20]: Do we need to uncomment this the first time we run the tests?

**Review by julianmclain — APPROVED** [review]

File: test/helper.js (hunk 1 above)
Comment (francisli) [reply test/helper.js:20]: No, good catch, I'll remove this

**Pull request merged**